lib/Secrets/Plugins/extensionplugins.h

```diff
@@ -35,6 +35,8 @@ class SAILFISH_SECRETS_API PluginBase
     PluginBase();
     virtual ~PluginBase();
 
+    virtual void initialize();
```

Comment:
I thought this could return a bool indicating success/failure, and a failed plugin could be removed. But I guess it makes a sort of sense to keep them around to return meaningful errors if someone tries to utilize the plugin.